### PR TITLE
[visual-basic] Replace ~/ links with ../ relative links

### PR DESCRIPTION
**Pull request by nemrism**
Replace `~/` links with `../` relative links in /visual-basic/** files

Contributes to #9968

### CODE REVIEW

File: docs/visual-basic/programming-guide/concepts/linq/how-to-filter-on-an-attribute-xpath-linq-to-xml.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This topic shows how to get the descendant elements with a specified name, and w
   
  The XPath expression is:  
   
- `.//Address[@Type='Shipping']`  
+ `./Address[@Type='Shipping']`
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-filter-on-an-attribute-xpath-linq-to-xml.md]: this is not a path change. Why is this included here?

File: docs/visual-basic/programming-guide/concepts/linq/how-to-find-related-elements-xpath-linq-to-xml.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ This topic shows how to get an element selecting on an attribute that is referre
   
  The XPath expression is:  
   
- `.//Customer[@CustomerID=/Root/Orders/Order[12]/CustomerID]`  
+ `./Customer[@CustomerID=/Root/Orders/Order[12]/CustomerID]`
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/how-to-find-related-elements-xpath-linq-to-xml.md]: ditto

File: docs/visual-basic/programming-guide/concepts/linq/statically-compiled-queries-linq-to-xml.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ Next
 Dim reader = Xml.XmlReader.Create("PurchaseOrders.xml")  
 Dim doc As New Xml.XmlDocument()  
 doc.Load(reader)  
-Dim nl As Xml.XmlNodeList = doc.SelectNodes(".//Address[@Type='Shipping']")  
+Dim nl As Xml.XmlNodeList = doc.SelectNodes("./Address[@Type='Shipping']")
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/linq/statically-compiled-queries-linq-to-xml.md]: ditto

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @nemrism! The changes look good overall but I don't understand why some files with changes not related to relative links were included.

File: docs/visual-basic/programming-guide/concepts/linq/how-to-filter-on-an-attribute-xpath-linq-to-xml.md (hunk 1 above)
Comment (nemrism) [reply docs/visual-basic/programming-guide/concepts/linq/how-to-filter-on-an-attribute-xpath-linq-to-xml.md]: My bad... I was fixing a previous replace error with this. 
I'll correct this right now!

**Review by mairaw — APPROVED** [review]
Looks good now. Thanks!
